awscli/customizations/iot.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2015 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+"""
+This customization makes it easier to save various pieces of data
+returned from iot commands that would typically need to be saved to a
+file. This customization adds the following options:
+
+- aws iot create-certificate-from-csr
+  - ``--certificate-pem-outfile``: certificatePem
+- aws iot create-keys-and-certificate
+  - ``--certificate-pem-outfile``: certificatePem
+  - ``--public-key-outfile``: keyPair.PublicKey
+  - ``--private-key-outfile``: keyPair.PrivateKey
+"""
+from awscli.customizations.arguments import QueryOutFileArgument
+
+
+class IoTWrapper(object):
+
+    def __init__(self, event_handler):
+        self._cert_arg = QueryOutFileArgument(
```

Comment:
We are creating these arguments upon every command run, even if it is not iot related. Could we instantiate these in the arguments in the handler that gets registered to the events?